### PR TITLE
more changes relating to #65

### PR DESCRIPTION
**Pull request by cornernote**
batch generator should have options for ActiveQuery class

### CODE REVIEW

File: commands/BatchController.php [hunk 1]
```diff
@@ -104,6 +104,21 @@ class BatchController extends Controller
...
 
     /**
+     * @var bool indicates whether to generate ActiveQuery for the ActiveRecord class
+     */
+    public $generateQuery = true;
+
+    /**
+     * @var string the namespace of the ActiveQuery class to be generated
+     */
+    public $queryNs = 'app\models\query';
```
Comment (schmunk42) [inline commands/BatchController.php]: Should be `$modelQueryNs` in the BatchController

File: commands/BatchController.php [hunk 2]
```diff
@@ -192,7 +210,10 @@ public function actionModels()
                 'modelClass'         => isset($this->tableNameMap[$table]) ? $this->tableNameMap[$table] :
                     Inflector::camelize($table), // TODO: setting is not recognized in giiant
                 'baseClass'          => $this->modelBaseClass,
-                'tableNameMap'       => $this->tableNameMap
+                'tableNameMap'       => $this->tableNameMap,
+                'generateQuery'      => $this->generateQuery,
+                'queryNs'            => $this->queryNs,
```
Comment (schmunk42) [inline commands/BatchController.php]: Same here `queryNamespace`

**Comment by cornernote** [comment]
like this ?

**Comment by schmunk42** [comment]
Still can't merge.

**Comment by cornernote** [comment]
because of conflicts, or because I need to change something else?

I can't easily see the conflicts.

**Comment by schmunk42** [comment]
Just GitHub says so?!

**Comment by cornernote** [comment]
Its only 3 additions, no modification.  Can you just pull the change in by hand, or should I try a new PR?

**Pull request merged**